src/client/components/ServicesPage/ServicesPage.js

```diff
@@ -85,9 +85,10 @@ const renderServices = (services, loading, components) =>{
 						services.map( service =>{
 							service.vehicle = null; /** Setting it to null because no point in repeating the vehicle name in the services page */
 							service.date = prettifyDate(service.date);
+							const serviceId = service.component;
```

Comment:
**service.component** returns strings like **engine-oil** etc. Also it can repeat for services. I think for using as key, you could use **service._id** which is an actual key like unique string.